content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,291 @@
+              fieldRef:
+                fieldPath: metadata.labels['tags.datadoghq.com/env']
+          - name: DD_SERVICE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.labels['tags.datadoghq.com/service']
+          - name: DD_VERSION
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.labels['tags.datadoghq.com/version']
+```
+
+##### Specific configuration
+
+###### Pod-level metrics
+Add the following standard labels (`tags.datadoghq.com`) to the pod spec of a Deployment, StatefulSet, or Job:
+
+```yaml
+  template:
+    metadata:
+      labels:
+        tags.datadoghq.com/env: <ENV>
+        tags.datadoghq.com/service: <SERVICE>
+        tags.datadoghq.com/version: <VERSION>
+```
+These labels cover pod-level Kubernetes CPU, memory, network, and disk metrics, and are used for injecting `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` into your service's container.
+
+If you have multiple containers per pod, you can specify standard labels by container:
+
+```yaml
+tags.datadoghq.com/<container-name>.env
+tags.datadoghq.com/<container-name>.service
+tags.datadoghq.com/<container-name>.version
+```
+
+###### State metrics
+
+For [Kubernetes State Metrics][2], add the same standard labels to the collection of labels for the parent resource (e.g., Deployment).
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    tags.datadoghq.com/env: <ENV>
+    tags.datadoghq.com/service: <SERVICE>
+    tags.datadoghq.com/version: <VERSION>
+spec:
+  <Block from above>
+```
+
+###### APM Tracer / StatsD client
+
+[APM Tracer][3] and [StatsD client][4] environment variables can be sourced from the same labels by using the [Kubernetes's downward API][5] in the format below:
+
+```yaml
+    containers:
+    -  ...
+       env:
+            - name: DD_ENV
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/env']
+            - name: DD_SERVICE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/service']
+            - name: DD_VERSION
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/version']
+```
+
+[1]: /agent/faq/auto_conf/
+[2]: /agent/kubernetes/data_collected/#kube-state-metrics
+[3]: /tracing/send_traces/
+[4]: /integrations/statsd/
+[5]: https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api
+{{% /tab %}}
+
+{{% tab "Docker" %}}
+
+To achieve a single point of configuration across all telemetry, make `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` available within your container's runtime environment. As long as the agent can access your container's Docker socket, it will be able to detect the environment variables and map them to the standard tags. These variables can be set in the Docker image:
```

Comment:
One thing occurred to me: technically this is only true if folks use logs injection.

Otherwise their logs won't properly get a `service` attribute. They'd either need:
```
com.datadoghq.tags.service: <service>
```
or
```
com.datadoghq.logs: '{"service": "<SERVICE>", ...}'
```

We might want to add a note encouraging logs injection (especially for APM folks, who might not be benefitting from that connection yet). If logs injection isn't desired, we can then say that there is a couple of options available.